tools/clang/test/CodeGenSPIRV/rich.debug.type.composite.hlsl

```diff
@@ -30,15 +30,15 @@ struct foo {
 // CHECK: [[func1:%\d+]] = OpString "foo.func1"
 // CHECK: [[func0:%\d+]] = OpString "foo.func0"
 
-// CHECK: [[parent:%\d+]] = OpExtInst %void [[set]] DebugTypeComposite [[foo]] Structure {{%\d+}} 3 1 {{%\d+}} {{%\d+}} %uint_192 FlagIsProtected|FlagIsPrivate [[a:%\d+]] [[f0:%\d+]] [[b:%\d+]] [[f1:%\d+]] [[c:%\d+]]
 // CHECK: [[bool:%\d+]] = OpExtInst %void %1 DebugTypeBasic [[bool_name]] %uint_32 Boolean
+// CHECK: [[parent:%\d+]] = OpExtInst %void [[set]] DebugTypeComposite [[foo]] Structure {{%\d+}} 3 8 {{%\d+}} {{%\d+}} %uint_0 FlagIsProtected|FlagIsPrivate [[a:%\d+]] [[b:%\d+]] [[c:%\d+]] [[f0:%\d+]] [[f1:%\d+]]
 
-// CHECK: [[c]] = OpExtInst %void [[set]] DebugTypeMember [[c_name]] [[bool]] {{%\d+}} 19 3 [[parent]] %uint_160 %uint_32 FlagIsProtected|FlagIsPrivate
+// CHECK: [[c]] = OpExtInst %void [[set]] DebugTypeMember [[c_name]] [[bool]] {{%\d+}} 19 8 [[parent]] %uint_0 %uint_0 FlagIsProtected|FlagIsPrivate
```

Comment:
It is because of the unknown physical layout of `foo a;` which is a local variable.

When we have to specify the `size` and the `offset` of a type,
- If we know the physical layout, we will use the physical layout information. Note that the same type can have different size/offset depending on its physical layout or storage class.
- If we do not know the physical layout, currently there is no way to express it. For example, we do not know the physical layout for local variables. I temporarily set it zero. We discussed this issue in the SPIR-V spec issue https://gitlab.khronos.org/spirv/SPIR-V/-/issues/578. We will soon update the spec to support variables with the unknown physical layout.